add TiFlash spill doc (#13342)
pingcap/docs-cn: tiflash/tiflash-configuration.md

    ## 从 v6.2.0 引入，表示 PageStorage 单个数据文件中有效数据的最低比例。当某个数据文件的有效数据比例低于该值时，会触发 GC 对该文件的数据进行整理。默认为 0.5。
    dt_page_gc_threshold = 0.5

>     ## 从 v7.0.0 引入，表示带 group by key 的 HashAggregation 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashAggregation 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。

Reviewer: ```suggestion
    ## 从 v7.0.0 引入，表示带 group by key 的 HashAggregation 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashAggregation 会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制。
```

    max_bytes_before_external_group_by = 0

>     ## 从 v7.0.0 引入，表示 sort/topN 算子在触发 spill 之前的最大可用内存，超过该阈值之后 sort/TopN 会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。

Reviewer: ```suggestion
    ## 从 v7.0.0 引入，表示 sort/topN 算子在触发 spill 之前的最大可用内存，超过该阈值之后 sort/TopN 会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制。
```

    max_bytes_before_external_sort = 0

>     ## 从 v7.0.0 引入，表示带等值 join 条件的 HashJoin 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashJoin 算子会采用 spill to disk 的方式来减小内存使用。配置成 0 表示内存使用无限制，默认为 0。

Reviewer: ```suggestion
    ## 从 v7.0.0 引入，表示带等值 join 条件的 HashJoin 算子在触发 spill 之前的最大可用内存，超过该阈值之后 HashJoin 算子会采用 spill to disk 的方式来减小内存使用。默认值为 0，表示内存使用无限制。
```

>     max_bytes_before_external_sort = 0

Reviewer: `max_bytes_before_external_sort = 0` is the same as line 207. Please check.

Author: Should be `max_bytes_before_external_join`
